docs/how-to-run-examples.md

```diff
@@ -0,0 +1,80 @@
+# How to run examples
+
+# Preparing nntrainer for execution.
+## Use PPA
+
+If you don't mind building binaries, you can directly download from PPA with daily releases:
+* Download nntrainer :
+```
+$ sudo add-apt-repository ppa:nnstreamer/ppa
+$ sudo apt-get update
+$ sudo apt-get install nntrainer
```

Comment:
ppa:nnstreamer/ppa provides Bionic only. please add the Xenial.

Reply:
yes. we will. Thanks.